appcelerator/titanium_mobile
# [TIMOB-25753] Android: Fixed bug where ScrollView "width" ignores Ti.UI.SIZE as of 6.3.0

**JIRA:** https://jira.appcelerator.org/browse/TIMOB-25753

**Test:**

1. Run test attached to [TIMOB-25753](https://jira.appcelerator.org/browse/TIMOB-25753) on Android.
2. Verify ScrollView is shrunk width-wise to just fit the label as shown in [TIMOB-25753](https://jira.appcelerator.org/browse/TIMOB-25753) attached `Screenshot-Good.png`.
